packages/@ember/-internals/glimmer/lib/syntax/outlet.ts

```diff
@@ -38,27 +38,18 @@ import { OutletState } from '../utils/outlet';
   Note: Your content __will not render__ if there isn't an `{{outlet}}` for it.
 
   @method outlet
-  @param {String} [name]
   @for Ember.Templates.helpers
   @public
 */
 export const outletHelper = internalHelper(
-  (args: CapturedArguments, owner?: Owner, scope?: DynamicScope) => {
+  (_args: CapturedArguments, owner?: Owner, scope?: DynamicScope) => {
     assert('Expected owner to be present, {{outlet}} requires an owner', owner);
     assert(
       'Expected dynamic scope to be present. You may have attempted to use the {{outlet}} keyword dynamically. This keyword cannot be used dynamically.',
       scope
     );
-    let nameRef: Reference<string>;
-
-    if (args.positional.length === 0) {
-      nameRef = createPrimitiveRef('main');
-    } else {
-      let maybeNameRef = args.positional[0];
-      assert('Expected at least one positional arg', maybeNameRef);
-      nameRef = maybeNameRef;
-    }
 
+    let nameRef: Reference<string> = createPrimitiveRef('main');
```

Comment:
I think this should go too. Below, `outlets[valueForRef(nameRef)]` becomes `outlets.main`

Reply:
Removed. Thanks!